src/coreclr/src/vm/frames.h

```diff
@@ -945,6 +945,8 @@ class TransitionFrame : public Frame
         return FALSE;
     }
 
+    virtual MethodDesc* GetCanonicalFunction();
```

Comment:
This does not need to be virtual method.

Reply:
Removed virtual